packages/squiggle-lang/src/rescript/Distributions/DistributionTypes.res

```diff
@@ -91,7 +91,7 @@ module DistributionOperation = {
     | ToString
     | ToSparkline(int)
 
-  type toScore = KLDivergence(genericDist)
+  type toScore = KLDivergence(genericDist) | LogScore(float, option<genericDist>)
```

Comment:
This naming is pretty confusing, because KLDivergence is itself a log score. However, we can focus on this later, as this part of the codebase will likely be getting refactored later anyway.